apis/zora/v1alpha1/clusterscan_types.go

```diff
@@ -39,6 +39,18 @@ type ClusterScanSpec struct {
 
 	// The list of Plugin references that are used to scan the referenced Cluster.  Defaults to 'popeye'
 	Plugins []PluginReference `json:"plugins,omitempty"`
+
+	// SuccessfulScansHistoryLimit specifies the amount of successfully
+	// completed scan Jobs to be kept in the cluster. This field is analogous
+	// to <Cronjob.Spec.SuccessfulJobsHistoryLimit> from the <batch> package.
+	// +kubebuilder:validation:Minimum=0
+	SuccessfulScansHistoryLimit *int32 `json:"successfulScansHistoryLimit,omitempty"`
```

Comment:
We need a default value for this field, so I recommend add this marker: `// +kubebuilder:default=3`

Reply:
Done